src/screen/GameScreen.java

```diff
@@ -623,6 +654,11 @@ && checkCollision(bullet, this.enemyShipSpecial)) {
 					recyclable.add(bullet);
 				}
 
+				if (this.itemManager.getShotNum() == 1 && bullet.getPositionY() < topEnemyY) {
+					this.combo = 0;
+					isExecuted = true;
+				}
+
```

Comment:
Could you tell me why this changes made?

Reply:
When bullets are multiplyed, combo is initalized in case one of them is out of top. So when using item, we decided not applying that method.